src/lib/crud.service.ts

```diff
@@ -162,4 +172,14 @@ export class CrudService<T extends BaseEntity> {
             await runner.release();
         }
     }
+
+    private excludeEntity(entity: T, exclude: Set<string>): T {
+        if (exclude.size === 0) {
+            return entity;
+        }
+        for (const excludeKey of exclude.values()) {
+            delete entity[excludeKey as unknown as keyof T];
+        }
+        return entity;
+    }
```

Comment:
I'm quite afraid of calling `delete` operator so many times affects performance.
When developer wants to exclude a number of fields from an entity, `delete` operation will be executed multiple times in each object. It could be even worse in 'readMany' or 'search' method. 😱

Reply:
I wish we could do this on TypeORM level, when we selecting records from the table 🤔

Reply:
or how about set value of those exclude key to `undefined`.

Reply:
right, I am concerned about the double loop in readMany.
I think it will be fine in other methods.
Do you have any good ideas? 🤔

Reply:
ok, wait, plz...

Reply:
Changed readOne and readMany use select clause